caps/eth.md

```diff
@@ -108,22 +108,173 @@ relayed by the peer.
 Transactions must be validated before re-propagating them. Relaying an invalid transaction
 results in peer disconnection.
 
+### Transaction Encoding and Validity
+
+Transaction objects exchanged by peers have one of two encodings, which are identical to
+the encodings used by Ethereum consensus. In definitions across this specification, we
+refer to transaction objects of either encoding using the identifier `txₙ`.
+
+    tx = {legacy-tx, typed-tx}
+
+Untyped, legacy transactions are given as an RLP list.
+
+    legacy-tx = [
+        nonce: P,
+        gas-price: P,
+        gas-limit: P,
+        recipient: {B_0, B_20},
+        value: P,
+        data: B,
+        V: P,
+        R: P,
+        S: P,
+    ]
+
+[EIP-2718] typed transactions are encoded as RLP byte arrays where the first byte is the
+transaction type (`tx-type`) and the remaining bytes are opaque type-specific data.
+
+    typed-tx = rlp(tx-type || tx-data)
+
+Transactions must be validated when they are received. Validity depends on the Ethereum
+chain state. While the encoding of typed transactions is opaque, it is assumed that the
+encoding of `tx-data` provides values for `nonce`, `gas-price`, `gas-limit`, and that the
+sender account of the transaction can be determined from the signature.
+
+Transactions must be validated according to these rules:
+
+- If the transaction is typed, the `tx-type` must be one of the types considered
+  acceptable at the current block height.
+- The signature (`V`, `R`, `S` values) must be valid according to the signature schemes
+  supported by the chain. At this time, the two schemes in active use are the basic
+  'Homestead' scheme where `V`, `R`, `S` are simply the components of a secp256k1
+  signature; and the [EIP-155] scheme, where the signature values include a 'chain ID' to
+  prevent replay of the transaction on a different chain.
```

Comment:
I recommend removing this line from the network specification, or greatly curbing its prescriptiveness to something like:
"If the transaction type has a signature, then it must be validated upon receipt of the transaction before propagating it to peers."

EIP-2718 makes no statements about the signing scheme, it is up to each individual transaction type to define that.  In fact, there may be some transaction types in the future that have no signature associated with them.  EIP-2930, for example, uses a different signing scheme than legacy transactions (`yParity` instead of `v`).